tests/test_loaders.py

```diff
@@ -164,25 +164,25 @@ def test_download(mocker):
 
 # This is magically skipped by the the remote fixture `skip_local` in conftest.py
 # when tests are run with the --local flag
-def test_validate(skip_local):
-    for dataset_name in DATASETS:
-        data_home = os.path.join("tests/resources/mir_datasets", dataset_name)
-        dataset = mirdata.Dataset(dataset_name, data_home=data_home)
-        try:
-            dataset.validate()
-        except:
-            assert False, "{}: {}".format(dataset_name, sys.exc_info()[0])
-
-        try:
-            dataset.validate(verbose=False)
-        except:
-            assert False, "{}: {}".format(dataset_name, sys.exc_info()[0])
-
-        dataset_default = mirdata.Dataset(dataset_name, data_home=None)
-        try:
-            dataset_default.validate(verbose=False)
-        except:
-            assert False, "{}: {}".format(dataset_name, sys.exc_info()[0])
+# def test_validate(skip_local):
```

Comment:
Why are these commented? Note that if they're taking time to run when you do it locally, you can use the `--local` flag when running the tests to skip tests like this

Reply:
Yes, I know. But I had a problem with circleCI, and I was checking what was wrong.